### ISSUE TITLE
Search: remaining adjustments

### ISSUE THREAD

**Issue by peterfremlin** [peterfremlin/Disability-Debrief#37]
- [x] search over tagged categories/countries

There are three links in Zimbabwe/health. But when I search for "[zimbabwe health](https://www.disabilitydebrief.org/search/?ft=&fc=&q=zimbabwe+health)" (in main search bar, without filters) it only gives two, because one of them is from zimbabwe being a tagged country, rather than the principal country.

It would be good if that search returned all three.

- [x] In library link results, the tag to region gives the wrong link

e.g. searching for "Mapping and Recommendations on Disability-Inclusive Education" the library link result is tagged to Africa and the url to /library/regionafrica. I'm not sure why this isn't country-africa - but either way region-africa is the correct url.

If that's a problem on my json side, let me know?

- [x] saved settings don't need cookies, just a "copy saved search line"

Thanks for making it more functional, but I prefer simpler, just a line that says "Copy a link to this search."

- [x] trailing whitespace in library link results

for example the placement of "zimbabwe" in this should be directly after preceeding text

<img width="624" alt="Screenshot 2024-05-02 at 14 27 41" src="https://github.com/peterfremlin/Disability-Debrief/assets/106767665/065f7760-e5df-4db1-9caf-b37b862f85ba">

This could well be trailing whitespace from my json, so let me know if needs to be fixed my side

- [x] Pages for Regions should have the same globe by it as countries

eg by Asia, here:

<img width="418" alt="Screenshot 2024-05-02 at 14 35 15" src="https://github.com/peterfremlin/Disability-Debrief/assets/106767665/550c9110-5bb1-49a3-9ab7-85d44b49b552">

- [ ] bug with tags in search results (possibly International/Global related?)

**Comment by cathysarisky** [comment]
I need some help understanding this: "There are three links in Zimbabwe/health. But when I search for "[zimbabwe health](https://www.disabilitydebrief.org/search/?ft=&fc=&q=zimbabwe+health)" (in main search bar, without filters) it only gives two, because one of them is from zimbabwe being a tagged country, rather than the principal country." 

What's a 'tagged country'?  Am I missing something from your data structure?

**Comment by cathysarisky** [comment]
> I need some help understanding this: "There are three links in Zimbabwe/health. But when I search for "[zimbabwe health](https://www.disabilitydebrief.org/search/?ft=&fc=&q=zimbabwe+health)" (in main search bar, without filters) it only gives two, because one of them is from zimbabwe being a tagged country, rather than the principal country."
> 
> What's a 'tagged country'? Am I missing something from your data structure?

I /think/ I've improved the results when no filters are chosen.  I'm still not sure I exactly understand this request, so please take a look.  (Database changes are live on site.  Display changes are not.)

**Comment by cathysarisky** [comment]
![image](https://github.com/peterfremlin/Disability-Debrief/assets/42299862/23e12cf1-e21f-4509-b3fc-4d00c8a8be26)

See above for revision on handling regions.  Input/feedback welcome.  It feels a little weird to list Africa > Africa, but that's how your database is actually set up, I guess, so...

**Comment by peterfremlin** [comment]
Most this is looking great however there is a problem on the link some country/region tags in library link results.

e.g. if I search for "Mapping and Recommendations on Disability-Inclusive Education" the library link result is tagged "africa" but clicking that leads to an /undefined/. It should go to country-africa

e.g. if I search for "disabled body ages" then it is "international news" and that should go to country-int

**Comment by cathysarisky** [comment]
Thanks. I'll get the undefined link corrected.

Second bullet point:  I'm sorry, I've lost the context here.  Did you decide you didn't want pages shown?  

![image](https://github.com/user-attachments/assets/d5f4f202-ead4-4912-adbc-2bdea242f177)

**Comment by peterfremlin** [comment]
Ah sorry my mistake I'd forgotten about the pages results! Please keep those. But adjust the order of announcement to match the order they are on the page itself

**Comment by cathysarisky** [comment]
agreed, it's weird that the order doesn't match!  So... we like the order the results are displayed in, but the summary at the top should match it?  (Or we like the order of the summary, and please adjust the results?)

**Comment by peterfremlin** [comment]
We like the order results are displayed, change the summary to match

**Comment by peterfremlin** [comment]
The order looks good but this issue remains:

there is a problem on the link some country/region tags in library link results.

- e.g. if I search for "Mapping and Recommendations on Disability-Inclusive Education" the library link result is tagged "africa" but clicking that leads to an /undefined/. It should go to country-africa

- e.g. if I search for "disabled body ages" then it is "international news" and that should go to country-int

And in fact, trying to click on some of the country tags on the end of the "library" are all broken now? they all go to https://www.disabilitydebrief.org/search/undefined/

**Comment by cathysarisky** [comment]
Ah. Possibly the result of a combintion of "International News" and "Global" labels.  Fixed a bug, added some fallback behavior.  Will roll out a new version of theme to you shortly.

**Comment by peterfremlin** [comment]
It's working better but not consistently everywhere. For example clicking international news on this search goes to undefined

https://www.disabilitydebrief.org/search/?ft=&fc=&q=migrants+disabilities+dual+neglect

**Comment by peterfremlin** [comment]
Nice one, works great now!
